tests/modules/programs/newsboat/default.nix

```
@@ -1,4 +1,5 @@
 {
   newsboat-basics = ./newsboat-basics.nix;
   newsboat-basics-2003 = ./newsboat-basics-2003.nix;
+  newsboat-basics-2009 = ./newsboat-basics-2109.nix;
```

Shouldn't it be
```suggestion
  newsboat-basics-2105 = ./newsboat-basics-2105.nix;
```
?

---

Yep, poor copy-paste job from what you mentioned on the other PR: https://github.com/nix-community/home-manager/pull/1331#issuecomment-644334078

Should be fixed now.